Updates dependencies and ups version to 0.0.10

nigelmegitt opened the pull request:
Tracking pull request, addresses vulnerabilities

nigelmegitt committed:
Updates dependencies and ups version to 0.0.10

nigelmegitt committed:
Update dependencies

* `babel-plugin-istanbul` `6.0.0 -> 6.1.1`
* `webpack-cli` `4.9.0 -> 4.9.1`

danielthepope reviewed (commented):
I followed the Setup instructions in the README.

When I run `npm install`, I get the message
```
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
```
After it has installed, the lockfileVersion gets set to `1`, so perhaps something isn't documented correctly?

`npm --version` gives me `6.14.11`, which is the one installed when I ran `nvm install 14.16.0`. Perhaps you also need to add how to update NPM? I think it's `npm install -g npm`

nigelmegitt committed:
Track dependency updates

* `babel-loader` `8.2.2 -> 8.2.3`
* `webpack` `5.58.2 -> 5.59.1`

nigelmegitt committed:
Update node version to 14.8.1

Most recent LTS version of node. Hopefully addresses @danielthepope 's review comment.

nigelmegitt committed:
Update to npm 8.1.1

nigelmegitt committed:
Specify min node version in `package.json`

nigelmegitt committed:
Update build scripts for nvm and npm versions

nigelmegitt commented:
This has been overtaken by events (and more recent dependency versions). Closing without merging.
